mmtbx/tls/optimise_amplitudes.cpp

```diff
@@ -368,14 +368,14 @@ void MultiGroupMultiDatasetUijAmplitudeFunctionalAndGradientCalculator::calculat
 
 void MultiGroupMultiDatasetUijAmplitudeFunctionalAndGradientCalculator::calculateFGLeastSquares()
 {
-  // Normalisation term (number of datasets)
-  double norm_all = 1. / (double)(n_dst * n_atm);
+  // Normalisation term (none)
+  double norm_all = 1.;
```

Comment:
Only comment would be to remove `norm_all` if you're just multiplying by 1. But if you're planning on trying another normalization procedure, then keeping it is fine.